packages/material-ui/src/utils/useControlled.test.js

```diff
@@ -83,4 +83,11 @@ describe('useControlled', () => {
       'Material-UI: A component is changing the default value state of an uncontrolled TestComponent after being initialized.',
     );
   });
+
+  it('should not raise a warning if changing the defaultValue when controlled', () => {
+    const { setProps } = render(<TestComponent value={[]} defaultValue={[]}>{() => null}</TestComponent>);
+    expect(consoleErrorMock.callCount()).to.equal(0);
+    setProps({ value: null, defaultValue: null });
```

Comment:
Why do we need to change the `value` here as well? The linked issue only talks about changing `defaultValue`.

Reply:
Because I am modifying multiple and value at the same time in autocomplete, the error occurs.
do I need to delete it?
(Sorry, both my test ability and English ability are beginners, please understand)